src/potree.ts

```diff
@@ -305,14 +329,14 @@ export class Potree implements IPotree {
 
     return (
       pointClouds: PointCloudOctree[],
-      camera: Camera,
+      cameras: Camera[],
     ): {
-      frustums: Frustum[];
-      cameraPositions: Vector3[];
+      frustums: Frustum[][];
+      cameraPositions: Vector3[][];
       priorityQueue: BinaryHeap<QueueItem>;
     } => {
-      const frustums: Frustum[] = [];
-      const cameraPositions = [];
+      const frustums: Frustum[][] = [];
```

Comment:
I am a bit worried about the garbage due to creating all these arrays each frame.